src/language.js

```diff
@@ -203,7 +203,17 @@ const loadingIndicator = {
 
 const createNewOptionModal = {
   addNewAttributeTitle: (attribute) => `Add New ${attribute}`,
+  genusNotFound: () => (
+    <>
+      Genus not found in MERMAID
+      <br />
+      <a target="_blank" href="https://datamermaid.org/contact-us">
```

Comment:
Technically, the language file should have no logic other than some minor html for formatting, (bold italic, spans with class names). 

This is fine for now though. Someone can change it later.

Reply:
What's your recommendation for doing this the correct way?